src/ProjectTemplates/Web.ProjectTemplates/content/Api-CSharp/Program.cs

```diff
@@ -1,7 +1,6 @@
 #if (NativeAot)
 using System.Text.Json.Serialization;
 #endif
-using Company.ApiApplication1;
 
```

Comment:
Can you move this blank line to inside the `#if (NativeAot)` section? That way the file doesn't start with an empty line when you don't pass `--aot`.